ui/raidboss/data/02-arr/raid/t13.js

```diff
@@ -178,17 +178,17 @@
         'Akh Morn': 'Akh Morn',
         'Blood Add': 'Add Sang',
         'Dark Aether': 'éther sombre',
-        'Double Dive': 'Plongeon double',
+        'Double Dive': 'Double plongeon',
```

Comment:
What do you mean by "exact translation of skill names"? I think you are changing the skill name in game to something else, if I'm understanding these translations correctly:

https://xivapi.com/Action/3016?pretty=true
https://xivapi.com/Action/8568?pretty=true
https://xivapi.com/Action/8644?pretty=true
https://xivapi.com/Action/9734?pretty=true